src/makeCalculator.js

```diff
@@ -37,7 +37,64 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  let result = 0;
+
+  function add(num) {
+    result += num;
+  }
+
+  function subtract(num) {
```

Comment:
Make all functions be methods of object calculator
![image](https://user-images.githubusercontent.com/96409563/227483833-ec3325d8-6ae7-45d7-a6b4-ed9896794b11.png)